pkg/dashboard/resource/function.go

```diff
@@ -513,6 +513,10 @@ func (fr *functionResource) redeployFunction(request *http.Request,
 		return errors.Wrap(err, "Failed to get get function")
 	}
 
+	if function.GetConfig().Spec.Build.Image == "" {
+		return nuclio.NewErrPreconditionFailed("No image field in function config spec, unable to redeploy")
+	}
```

Comment:
Why? when you redeploy you need the "built" the image (spec.image) rather than the "to-build" which is spec.build.image

Reply:
@liranbg my bad, fixed